python/GafferSceneUI/PathFilterUI.py

```diff
@@ -143,9 +144,29 @@ def __dataMenu( self, vectorDataWidget, menuDefinition ) :
 			}
 		)
 
+	def __targetPlug( self, plug ) :
+
+		cellPlug = plug.ancestor( Gaffer.Spreadsheet.CellPlug )
+		if cellPlug is not None :
+			plug = plug.node()["out"][cellPlug.getName()]
```

Comment:
As in the other commit, I think this needs to go inside the loop (and the deque needs to be initialised with `plug` rather than `plug.outputs()`.

Reply:
Fixed in be98ad75784e4214aa624da56927089468194665